src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+function Category() {
+  return (
+    <div className="category">
+      <h2>Categories Page</h2>
```

Comment:
* As stated in the project requirement `Should display a button "Check status" only.` kindly add the button with the text `Check Status` as its a requirement [Check the requirement here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Should%20display%20a%20button%20%22Check%20status%22%20only.)

Reply:
Hi @Davidosky007, I've fixed it. Thank you.